src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -472,7 +473,7 @@ def create_image(resource_group_name, name, os_type=None, location=None,  # pyli
                  source_virtual_machine=None,
                  os_blob_uri=None, data_blob_uris=None,
                  os_snapshot=None, data_snapshots=None,
-                 os_disk=None, data_disks=None):
+                 os_disk=None, data_disks=None, tags=None):
```

Comment:
This is the method default value. It might be because of this? http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments